firedrake/mg/opencascade_mh.py

```diff
@@ -73,7 +73,7 @@ def make_coarse_mesh(stepfile, cad, element_size, dim, comm=COMM_WORLD, distribu
     curdir = os.path.dirname(stepfile) or os.getcwd()
     stepname = os.path.basename(os.path.splitext(stepfile)[0])
     geopath = os.path.join(curdir, "coarse-%s.geo" % (stepname))
-    mshpath = geopath.replace("geo", "msh")
+    mshpath = geopath.replace(".geo", ".msh")
```

Comment:
Why not `mshpath = os.path.join(curdir, "coarse-%s.msh" % stepname)` which is safer again?

Reply:
Good point!